bigquery/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>com.google.cloud.bigdataoss</groupId>
     <artifactId>bigdataoss-parent</artifactId>
-    <version>2.2.3-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
```

Comment:
I'm curious if that's required with `relativePath`?

Reply:
I think each component should have a version explicitly declared in Maven POM